src/Avatar/Avatar.js

```diff
@@ -24,6 +24,9 @@ const getInitials = name => {
   return firstInitial;
 };
 
+const getText = name =>
+  name.match(/[A-Za-z]/) ? getInitials(name).toUpperCase() : name;
```

Comment:
would it be better to make this switch on whether the passed name is a string or not, and update the proptypes to allow numbers as well?

Reply:
ignore above. i see that we're passing '+<number>' rather than just the number